projects/vdk-plugins/vdk-oracle/src/vdk/plugin/oracle/ingest_to_oracle.py

```diff
@@ -18,12 +20,90 @@
 log = logging.getLogger(__name__)
 
 
+# Functions for escaping special characters
+def _is_plain_identifier(identifier: str) -> bool:
+    class InvalidOracleIdentifierError(BaseException):
+        pass
+
+    # Raise error identifier it starts with a number
```

Comment:
Can't it start with a number of it's in double quotes??

Reply:
Theoretically it can't start with a number as stated here https://docs.oracle.com/en/error-help/db/ora-00904/

It worked when it started with a special character, though, so who knows. I'll test quickly and see what happens.